client/client-multi/client-cpp/CMakeLists.txt

```diff
@@ -221,7 +221,7 @@ endif()
 if(CMAKE_COMPILER_IS_GNUCXX)
     list(APPEND KAA_COMPILE_OPTIONS 
                     -Wall
-                    -std=c++11) 
```

Comment:
We should stick with the standard as much as possible. GNU extensions are banned unless we can't leave without them. It's ok to enable them for QNX only, though.